code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -76,6 +76,16 @@
 	new /obj/item/weapon/reagent_containers/hypospray/medipen(src)
 	return
 
+/obj/item/weapon/storage/box/security
+
+/obj/item/weapon/storage/box/security/New()
+	..()
+	contents = list()
+	new /obj/item/clothing/mask/gas/sechailer(src)

```

Comment:
The idea came up in game, and to be honest the only reason sec officers lost their iconic gas masks was because Paprika threw a fit about their apperance.